### PR TITLE
A very small fix to silence some warnings.

### PR DESCRIPTION
**Pull request by haberman**
Also updated the Gemfile.lock since alpha-4 has been
pushed to RubyGems.

Change-Id: I8ddc5f125f28aa9a33c88dfe48251a75a877e1d3

### CODE REVIEW

**Comment by haberman** [comment]
Review to @raggi.

**Comment by raggi** [comment]
LGTM

File: ruby/ext/google/protobuf_c/encode_decode.c [hunk 1]
```diff
@@ -35,11 +35,13 @@
...
 //   https://bugs.ruby-lang.org/issues/11328
 VALUE noleak_rb_str_cat(VALUE rb_str, const char *str, long len) {
+  char *p;
   size_t oldlen = RSTRING_LEN(rb_str);
   rb_str_modify_expand(rb_str, len);
-  char *p = RSTRING_PTR(rb_str);
+  p = RSTRING_PTR(rb_str);
   memcpy(p + oldlen, str, len);
   rb_str_set_len(rb_str, oldlen + len);
+  return rb_str;
```
Comment (raggi) [inline ruby/ext/google/protobuf_c/encode_decode.c]: technically this isn't a warning fix, this looks like a real semantic fix, no?

Comment (haberman) [inline ruby/ext/google/protobuf_c/encode_decode.c]: Depends on how you look at it -- the one callsite for this function does not use the return value.

Comment (raggi) [inline ruby/ext/google/protobuf_c/encode_decode.c]: indeed. it's more a pointless nit pick that's not worth worrying about :fast_forward:

**Pull request merged**